adviser/base/openshift-templates/advise-demo.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: advise-demo
```

Comment:
It might be a good idea to remove the `demo` part and substitute it with `repo-initialization` or something like that from the beginning.